<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NO_OF_PPL</class>
 <widget class="QDialog" name="NO_OF_PPL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>1221</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Please enter the required information</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="Done">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>500</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>1171</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>How many people are you?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NoPeople"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
